openapi/paths/ready-to-pay.yaml

```
@@ -3,21 +3,21 @@ post:
     - Core
   tags:
     - Transactions
-  summary: Ready to Pay
+  summary: Ready to pay
   operationId: PostReadyToPay
   x-sdk-operation-name: readyToPay
   description: |-
-    Get available payment methods for a specific transaction or a purchase.
+    Retrieves available payment methods for a specific transaction or purchase.
 
-    The payment methods order shown to a customer **SHOULD** be the same as the order in the response.
+    The order in which the payment methods are displayed to the customer must be the same as the order in the response.
```

```suggestion
    The order in which the payment methods are displayed to the customer should be the same as the order in the response.
```
This is how we implement it for hosted forms, but people may choose their own approach.